### PR TITLE
benchmark: remove redundant assignment and double import

### PR DESCRIPTION
**Pull request by vilhelmprytz**
Changes in `benchmark.py`.

Assigning `methods = methods` is redundant.
Module `platform` was imported with both `import` and `import from`, merged into one `import from`.

### CODE REVIEW

File: benchmark.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 import texttable
 import sys
 import codecs
-from platform import python_version_tuple
+from platform import python_version_tuple, platform
```
Comment (AdhityaG) [inline benchmark.py:11]: Please revert this particular change as it is inconsistent with the principle of lazy loading - as is followed elsewhere in this project. What that means simply is that modules/packages/libraries should be imported as and when needed, and 'platform' is not needed at this line of code (it is used only within the 'benchmark' function)

File: benchmark.py [hunk 2]
```diff
@@ -110,9 +108,7 @@ def benchmark(n):
         results, ["Table formatter", "time, μs", "rel. time"], "rst", floatfmt=".1f"
     )
 
-    import platform
```
Comment (AdhityaG) [inline benchmark.py:113]: Also, revert this particular change as it is inconsistent with the general style followed elsewhere in this project. Revert the change and replace it with 'from platform import platform'

File: benchmark.py (hunk 1 above)
Comment (VilhelmPrytz) [reply benchmark.py:11]: Alright, I'll fix it and push tomorrow. May I ask why you have decided to lazy load, it's quite unusual to do so (generally most python linters warn when imports are not on top)?

**Comment by VilhelmPrytz** [comment]
@AdhityaG This has now been fixed.

**Comment by astanin** [comment]
That's all very interesting. Is there an advantage of doing one way or the other? It's a script which has to be run by literally one person (me), once or twice per year... 🤷‍♂️

**Comment by vilhelmprytz** [comment]
I might be wrong but with this change, it only has to import what it's looking for? I see why it is slightly unnecessary since the script is so rarely used but I don't see why it shouldn't be merged :slightly_smiling_face:

**Comment by astanin** [comment]
I'm convinced. Let's try it 😉
